Chummer/Backend/Uniques/MartialArtTechnique.cs

```diff
@@ -61,6 +62,11 @@ public void Create(XmlNode xmlTechniqueDataNode)
             if (xmlTechniqueDataNode.TryGetStringFieldQuickly("name", ref _strName))
                 if (!xmlTechniqueDataNode.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                     xmlTechniqueDataNode.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
+
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            xmlTechniqueDataNode.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in the `Create` method? Would there need to be data-definable colors for specific martial arts techniques?